tests/integration/targets/dcnm_inventory/tests/dcnm/merged.yaml

```diff
@@ -188,6 +188,43 @@
     fabric: "{{ ansible_it_fabric }}"
     state: deleted
 
+- name: MERGED - Merge a Switch using GreenField Deployment with non zero max hop
+  cisco.dcnm.dcnm_inventory: &conf_hop
+    fabric: "{{ ansible_it_fabric }}"
+    state: merged
+    config:
+    - seed_ip: "{{ ansible_switch1 }}"
+      auth_proto: MD5
+      user_name: "{{ switch_username }}"
+      password: "{{ switch_password }}"
+      max_hops: 2
```

Comment:
Update documentation to indicate this will always be set internally to 0 and update the code to set this to 0 regardless of what the user passes in.